server/database.sql

```diff
@@ -18,28 +18,104 @@ USE `cl_judge`;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
 --
--- Table structure for table `user`
+-- Table structure for table `groups`
 --
 
-DROP TABLE IF EXISTS `user`;
+DROP TABLE IF EXISTS `groups`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
-CREATE TABLE `user` (
+CREATE TABLE `groups` (
+  `id` varchar(45) NOT NULL,
+  `group_name` varchar(45) NOT NULL,
+  `confidential` tinyint NOT NULL,
+  `creator` varchar(45) NOT NULL,
+  `member_count` int NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `id_UNIQUE` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `notifications`
+--
+
+DROP TABLE IF EXISTS `notifications`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `notifications` (
+  `id` varchar(45) NOT NULL,
+  `heading` varchar(110) NOT NULL,
+  `description` varchar(110) DEFAULT NULL,
+  `created_at` varchar(110) NOT NULL,
+  `link` varchar(110) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `id_UNIQUE` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `user_groups`
+--
+
+DROP TABLE IF EXISTS `user_groups`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `user_groups` (
+  `id` varchar(45) NOT NULL,
+  `username` varchar(45) NOT NULL,
+  `group_id` varchar(45) NOT NULL,
+  `is_group_moderator` tinyint NOT NULL,
```

Comment:
`is_admin` changed to `is_group_moderator`

Reply:
great!